### PR TITLE
feat(ui): use workflow name and description

### LINKED ISSUE #933: Use workflow name & description
We show workflow description in the UI (which can be very long) and do not show workflow name anywhere.
We should use Workflow name (**and limit it's maximum number of characters**) and have description displayed in a different way.

CleanShot 2024-03-10 at 11 06 44@2x

### PR DESCRIPTION
**Pull request by pehlicd**
Closes #933 

## 📑 Description
Set workflow name as title of the workflow card and set 30 character limit.
Moved description to under the name with 50 character limit.

You can find how it looks like below:

keep-933

## ✅ Checks

- [x] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [ ] All the tests have passed

### CODE REVIEW

File: keep-ui/app/workflows/workflow-tile.tsx [hunk 1]
```diff
@@ -385,7 +385,7 @@ function WorkflowTile({ workflow }: { workflow: Workflow }) {
       )}
       <Card>
         <div className="flex w-full justify-between items-center h-14">
-          <Title>{workflow.description}</Title>
+          <Title>{workflow.name.length > 30 ? `${workflow.name.substring(0, 50)}...` : workflow.name}</Title>
```
Comment (talboren) [inline keep-ui/app/workflows/workflow-tile.tsx:388]: Use tailwind truncate for that is probably better

File: keep-ui/app/workflows/workflow-tile.tsx [hunk 2]
```diff
@@ -396,6 +396,10 @@ function WorkflowTile({ workflow }: { workflow: Workflow }) {
           })}
         </div>
 
+        <div className="flex w-fit justify-between items-center h-10">
+          <Text>{workflow.description.length > 50 ? `${workflow.description.substring(0, 50)}...` : workflow.description}</Text>
```
Comment (talboren) [inline keep-ui/app/workflows/workflow-tile.tsx:400]: Same as above

**Comment by shahargl** [comment]
is it ready to merge? @talboren

**Review by talboren — APPROVED** [review]
lgtm!

**Pull request merged**